manifests/install.pp

```diff
@@ -29,6 +29,7 @@
       command     => 'service apparmor reload',
       path        => ['/usr/local/sbin', '/usr/local/bin', '/usr/sbin', '/usr/bin', '/sbin', '/bin'],
       refreshonly => true,
+      onlyif      => '/usr/bin/test -d /sbin/apparmor_parser',
```

Comment:
Hang on ... `apparmor_parser` appears to be a file. Why the `-d`? Shouldn't this be `-f` or `-x`?